src/modules/game.js

```diff
@@ -63,7 +63,7 @@ export const displayScores = (list) => {
     str = '<li class="list-item">No scores</li>';
   } else {
     list.forEach((game) => {
-      str += `<li class="list-item">${game.user}: ${game.score}</li>`;
+      str += `<li class="list-item">${game.user}: <span>${game.score}</span></li><hr>`;
     });
     document.querySelector('.scores-list').innerHTML = str;
   }
```

Comment:
- [ optional] Great job with the board, it works pretty well. nevertheless, you skipped the point where the board is supposed to update only when the refresh button is clicked. onsubmit, the board automatically updates making the refresh button useless. kindly look into that